### PR TITLE
Add template for bug reports by QA team

### PR DESCRIPTION
**Pull request by matthew-white**
This PR adds an issue template that @srujner and @dbemke can use when filing issues. I modified the [Collect issue template](https://github.com/getodk/collect/blob/2d544e1a78174f0427b6f81bc35b6c7baac2c99d/.github/ISSUE_TEMPLATE.md). Suggestions welcome!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Please don't attach images of QR codes, as those provide access to the server.
+
+#### Expected behavior
+
+#### Central version shown in version.txt
+
+#### Browser version
+
+#### Around when did you see the problem (in UTC)?
```
Comment (matthew-white) [inline .github/ISSUE_TEMPLATE.md:23]: This is helpful to know when searching logs.

**Review by ktuite — APPROVED** [review]
Looks good to me! Was thinking screenshots _could_ include address bar, should we explicitly include or exclude that? Asking for the URL (and browser) directly seems most useful, and then it doesn't really matter what extra context is in the screenshot.

**Comment by matthew-white** [comment]
> Asking for the URL (and browser) directly seems most useful, and then it doesn't really matter what extra context is in the screenshot.

That sounds right to me. If the URL is in the text of the issue, then we can copy and paste it more easily. As long as the URL is in the text of the issue, I think it's OK either way whether the screenshot shows the URL or doesn't.

**Comment by srujner** [comment]
@matthew-white Everything looks good!

**Pull request merged**